openfisca_core/commons.py

```diff
@@ -1,5 +1,28 @@
 # -*- coding: utf-8 -*-
 
+# The following two variables and the is_unicode function are there to bridge string types across Python 2 & 3
+unicode_type = u"".__class__
+basestring_type = (str, unicode_type)
+
+
+def unicode_this(string, encoding=None):
```

Comment:
What about `to_unicode` ? The word `this` has a strong object-oriented connotation.

Reply:
Is this second parameter really necessary ? 
Why not encoding everything as `utf-8` ? I think it returns the same things for strings without special characters:

`unicode('2018')` and `unicode('2018', 'utf-8')` seem to be equal